src/common/threadpool.cpp

```diff
@@ -119,13 +118,12 @@ void threadpool::waiter::dec() {
 void threadpool::run() {
   boost::unique_lock<boost::mutex> lock(mutex);
   while (running) {
-    entry e;
     while(queue.empty() && running)
       has_work.wait(lock);
     if (!running) break;
 
     active++;
-    e = queue.front();
+    entry e = queue.front();
```

Comment:
not seeing the reason for this change either.

Reply:
same